CONTRIBUTING.adoc

```diff
@@ -273,7 +273,7 @@ We generate link:https://github.com/redhat-documentation/modular-docs[Modular Do
 
 Every time you submit a pull request a job will run that will verify the generation of modular documentation will succeed.
 
-Every time a pull request is merged to `main`, the modular documentation for the content in the `HEAD` of `main` will be generated to the link:https://github.com/bf2fc6cc711aee1a0c2a/guides/tree/modular-docs[modular docs] branch.
+Every time a pull request is merged to `main`, the modular documentation for the content in the `HEAD` of `main` will be generated to the link:https://github.com/bf2fc6cc711aee1a0c2a/app-services-guides/tree/modular-docs[modular docs] branch.
```

Comment:
Looks like the org needs updating too?

Reply:
We should probably future proof it by attributes